recipe/build.sh

```diff
@@ -1,29 +1,72 @@
-#!/bin/bash
+#!/usr/bin/env bash
+set -eu
```

Comment:
Almost always a good idea to use strict mode!

Reply:
Do you mean adding `o pipefail` and IFS?
`Makefile.cross` is in v5.4.0 - - Though, some changes are needed